test/integration/029_docs_generate_tests/test_docs_generate.py

```diff
@@ -574,7 +574,7 @@ def _verify_generic_macro_structure(self, manifest):
             if k not in {'macro_sql'}
         }
         # Windows means we can't hard-code these.
-        helpers_path = Normalized('macros/materializations/helpers.sql')
+        helpers_path = Normalized('macros/etc/column_helpers.sql')
```

Comment:
this scares me. should I be scared?

Reply:
No cause for fear. The comment a few lines up:
```
        # just test a known global macro to avoid having to update this every
        # time they change.
```
We want to verify that global/internal macros are included in `manifest.json`, and to do that we just test that one of them exists, with all of its properties, including its path.